template.go

```diff
@@ -29,52 +29,84 @@ func exists(path string) (bool, error) {
 	return false, err
 }
 
-func groupByMulti(entries []*RuntimeContainer, key, sep string) map[string][]*RuntimeContainer {
-	groups := make(map[string][]*RuntimeContainer)
-	for _, v := range entries {
-		value := deepGet(*v, key)
-		if value != nil {
-			items := strings.Split(value.(string), sep)
-			for _, item := range items {
-				groups[item] = append(groups[item], v)
-			}
+func getArrayValues(funcName string, entries interface{}) (reflect.Value, error) {
+	entriesVal := reflect.ValueOf(entries)
 
-		}
+	kind := entriesVal.Kind()
+
+	if kind == reflect.Ptr {
+		entriesVal = reflect.Indirect(entriesVal)
+		kind = entriesVal.Kind()
 	}
-	return groups
+
+	switch entriesVal.Kind() {
```

Comment:
Sorry I didn't notice this until after the merge, but shouldn't this be `switch kind {`?

Perhaps this can be the fix that gets put into #100, since I think it will have the same issue of returning an error when given a pointer to a slice or an array.